Add autoincrease of height of comment form (feature/68)

astronautr opened the pull request:
Additional work: refactor mass of code, cause code of all files related to postComment duplicated in issue view and controller.

astronautr committed:
Add autoincrease of height of comment form

Additional work: refactor mass of code, cause code of all files related to postComment duplicated in issue view and controller.

Card: #68

astronautr commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Как пользователь, хочу чтобы форма отправки комментария подстраивалась под количество текста, которое она содержит и автоматически расширялось/имело скролл.](https://trello.com/c/SyCq1hCv/68--)

kevromster commented on src/main/resources/static/assets/css/postComment.style.css:
```diff
@@ -0,0 +1,20 @@
+}
+.commentButton[type=button], input[type=submit] {
+    margin-left: 340px;
+}
```
пустая строка в конце

astronautr replied:
Done.

kevromster commented:
Не очень хорошо работает. Когда я третью строку начинаю писать, высота уменьшается:
![bug_increase_comment](https://cloud.githubusercontent.com/assets/16014197/20648717/12f8317c-b4e1-11e6-9dd8-5d67f4105947.png)
Потом обратно начинает увеличиваться, но некрасиво.
Проблема и в Chrome, и в Firefox.

astronautr committed:
Delete autoexpand for comment text area

Also change size of comment text area and move comment button to left side

Card: #68

astronautr committed:
Merge branch 'master' into feature/68

Card: #68
